### DISCUSSION TITLE
When setting `shell_integration: { osc9_9: true }` kitty sends a desktop notification everytime the working directory is changed

### DISCUSSION THREAD

**Discussion by Schweber** [nushell/nushell#13081] [Q&A]
Is this the way it's meant to be? Can i disable the notifications in `kitty.conf` or do i have to disable `osc9_9` integration if i don't want this behaviour?

**Comment by fdncred** [comment]
I know that osc9 is meant for notifications that some terminals respond to but osc9;9 shouldn't trigger a notification unless it's not supported by the terminal. Maybe kitty doesn't support osc9;9?

**Reply by Schweber** [reply]
https://github.com/nushell/nushell/issues/13092

**Reply by fdncred** [reply]
Kitty clearly has its own way to handle escapes. Not every terminal works like that. You have the ability to turn off osc9_9 which previously you didn't have, and it's turned off by default already.
